src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -18,34 +18,38 @@ New Relic's synthetic monitoring—powered by a Selenium-driven Chrome browser
 
 ### How to do it [#howto-1]
 
-1. To add a monitor, go to **[one.newrelic.com](http://one.newrelic.com) > Synthetics** (or [one.eu.newrelic.com](http://one.eu.newrelic.com) if you have an [EU-based account](/docs/using-new-relic/welcome-new-relic/get-started/introduction-eu-region-data-center)) and click **Create monitor**. Specify monitor type, name, and URL.
-2. Optional: Add a validation string (available for ping and simple browser) or advanced options, which enable substring monitoring for the following types of response validation:
+1. To add a monitor, go to **[one.newrelic.com](http://one.newrelic.com) > Synthetics** (or [one.eu.newrelic.com](http://one.eu.newrelic.com) if you have an [EU-based account](/docs/using-new-relic/welcome-new-relic/get-started/introduction-eu-region-data-center)) and click **Create monitor** at the top right of the page.
+    ![](/src/images/synthetics-create-monitor.png)
+2. Select a monitor type and then fill in all required fields.
+3. Optionally add tags, change the Period or Runtime (available for Simple browser, Scripted browser and API), or add a validation string (available for Ping and Simple browser) or advanced options, which enable substring monitoring for the following types of response validation:
 
    * **Verify SSL (for ping and simple browser).** This option verifies the validity of the SSL certificate chain. It can be duplicated by running the following syntax:
 
-     ```
+     ```sh
      openssl s_client -servername <var>{YOUR_HOSTNAME}</var> -connect <var>{YOUR_HOSTNAME</var>}:443 -CApath /etc/ssl/certs > /dev/null
      ```
-   * **Bypass HEAD request (for ping).** This option skips the default HEAD request and instead uses the GET verb with a ping check.
-   * **Redirect is Failure (for ping).** If a redirect result occurs when Redirect is Failure is enabled, Synthetics categorizes it as a failure (rather than following the redirect and checking the resulting URL).
-3. Select the locations where you want your monitor to run.
-4. Choose a frequency to determine how often each location will run your monitor.
-5. Optional: Set up alert notifications.
-6. Select **Create my monitor** to confirm.
-7. Wait a few minutes; then from the [Monitors index](#bookmark) check your monitor.
 
-## 2. View all monitors with the Monitors index page [#bookmark]
+   * **Bypass HEAD request (for Ping).** This option skips the default HEAD request and instead uses the GET verb with a ping check. GET requests will always happen if the HEAD request fails.
+   * **Redirect is Failure (for Ping).** If a redirect result occurs when Redirect is Failure is enabled, Synthetics categorizes it as a failure (rather than following the redirect and checking the resulting URL).
```

Comment:
```suggestion
   * **Redirect is Failure (for ping).** If a redirect result occurs when `Redirect is Failure` is enabled, synthetics categorizes it as a failure (rather than following the redirect and checking the resulting URL).
```